plugins/woocommerce/src/Admin/API/OnboardingThemes.php

```diff
@@ -184,6 +207,123 @@ public function activate_theme( $request ) {
+			 *
+			 * @since 8.3.0
+			 */
+			! apply_filters( 'woocommerce_allow_marketplace_suggestions', true ) ||
+			get_option( 'woocommerce_show_marketplace_suggestions', 'yes' ) === 'no'
+		) {
+			return array(
+				'themes' => array(),
+				'_links' => array(
+					'browse_all' => array(
+						'href' => home_url( '/wp-admin/themes.php' ),
+					),
+				),
+			);
+		}
+
+		// To be implemented: 1. Fetch themes from the marketplace API. 2. Convert prices to the requested currency.
+		// These are Dotcom themes.
+		$themes = array(
+			array(
+				'name'           => 'Tsubaki',
+				'price'          => 'Free',
+				'color_palettes' => array(),
+				'slug'           => 'tsubaki',
+				'is_active'      => false,
+				'thumbnail_url'  => 'https://i0.wp.com/s2.wp.com/wp-content/themes/premium/tsubaki/screenshot.png',
+				'link_url'       => 'https://wordpress.com/theme/tsubaki/',
+			),
+			array(
+				'name'           => 'Tazza',
+				'price'          => 'Free',
+				'color_palettes' => array(),
+				'slug'           => 'tazza',
+				'is_active'      => false,
+				'thumbnail_url'  => 'https://i0.wp.com/s2.wp.com/wp-content/themes/premium/tazza/screenshot.png',
+				'link_url'       => 'https://wordpress.com/theme/tazza/',
+			),
+			array(
+				'name'           => 'Amulet',
+				'price'          => 'Free',
+				'color_palettes' => array(),
+				'slug'           => 'amulet',
+				'is_active'      => false,
+				'thumbnail_url'  => 'https://i0.wp.com/s2.wp.com/wp-content/themes/premium/amulet/screenshot.png',
+				'link_url'       => 'https://wordpress.com/theme/tsubaki/',
+			),
+			array(
+				'name'           => 'Zaino',
+				'price'          => 'Free',
+				'color_palettes' => array(),
+				'slug'           => 'zaino',
+				'is_active'      => false,
+				'thumbnail_url'  => 'https://i0.wp.com/s2.wp.com/wp-content/themes/premium/zaino/screenshot.png',
+				'link_url'       => 'https://wordpress.com/theme/zaino/',
+			),
+		);
+
+		// To be implemented: Filter themes based on industry.
+		if ( $industry ) {
+			$filtered_themes = array_filter(
+				$themes,
+				function ( $theme ) use ( $industry ) {
+					// Filter themes by industry.
+					// Example: return $theme['industry'] === $industry;.
+					return true;
+				}
+			);
+		} else {
+			$filtered_themes = $themes;
+		}
+
+		$response = array(
+			'themes' => $filtered_themes,
+			'_links' => array(
+				'browse_all' => array(
+					'href' => home_url( '/wp-admin/themes.php' ),
+				),
+			),
+		);
+
+		/**
+		 * Filter the onboarding recommended themes response.
+		 *
+		 * @since 8.3.0
+		 *
+		 * @param array $response The recommended themes response.
+		 * @param array $filtered_themes The filtered themes.
+		 * @param string $industry The industry to filter by (if provided).
+		 * @param string $currency The currency to convert prices to. (USD, AUD, CAD, EUR, GBP).
+		 *
+		 * @return array
+		 */
+		return apply_filters(
+			'woocommerce_rest_get_recommended_themes',
```

Comment:
Could we make this an experimental filter for now?

```suggestion
			'__experimental_woocommerce_rest_get_recommended_themes',
```

Reply:
Sure! Updated in https://github.com/woocommerce/woocommerce/commit/fba3b839181446f8e1799bd0d6f45c1a1fb4ba2a.